src/Cohesity.Powershell/Scripts/Set-CohesityAlertResolutions.ps1

```diff
@@ -1,14 +1,45 @@
+function UpdateAlertResolution($alertIds, $resolvedAlertId) {
+    $url = $server + '/irisservices/api/v1/public/alertResolutions/'+$resolvedAlertId
+
+    $headers = @{'Authorization'='Bearer '+$token}
+    $payload = @{
+        alertIdList=@($alertIds)
+    }
+    $payloadJson = $payload  | ConvertTo-Json
+    $resp = Invoke-RestMethod -Method 'Put' -Uri $url -Headers $headers -Body $payloadJson -SkipCertificateCheck
+    $resolutionId = $resp.resolutionDetails.resolutionId
+    Write-Host "Successfully updated the resolution for the alert with id ="$resolutionId
+
+}
+function CreateAlertResolution($alertIds, $resolutionSummary, $resolutionDetails) {
```

Comment:
When we load the script ..it will convert all these functions into modules/cmdlets right ?
which is not safe ?  

https://github.com/cohesity/cohesity-powershell-module/blob/script-based/src/Cohesity.Powershell/Cohesity.PowerShell.psm1

Reply:
Fixed. Need to write modular components. Removing the additional functions and writing the inline codes. This is a temporary solution, need more reusable and sharable code in future.